service/matching/handler.go

```diff
@@ -362,6 +362,26 @@ func (h *Handler) ReplicateTaskQueueUserData(
 	return h.engine.ReplicateTaskQueueUserData(ctx, request)
 }
 
+// DispatchNexusTask implements matchingservice.MatchingServiceServer.
+func (h *Handler) DispatchNexusTask(ctx context.Context, request *matchingservice.DispatchNexusTaskRequest) (*matchingservice.DispatchNexusTaskResponse, error) {
+	panic("unimplemented")
```

Comment:
Should we just return an error instead of panic? Probably a non-issue but I'm just wondering if this code gets deployed and someone calls this endpoint will it cause the service to restart?

Reply:
There's an interceptor to catch the panics, but yes please return a proper error anyway

Reply:
I don't think we should worry about this at this point.
This PR is just for the API boilerplate.